target/linux/ath79/image/generic.mk

```diff
@@ -305,6 +305,16 @@ define Device/aruba_ap-105
 endef
 TARGET_DEVICES += aruba_ap-105
 
+define Device/atheros_db120
+  SOC := ar9344
+  DEVICE_VENDOR := Atheros
+  DEVICE_MODEL := DB120
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage
```

Comment:
From what I remember, we typically don't include usb-storage in default build.

Reply:
A number of devices do have usb-storage enabled (AP121F, Arduino Yun, EG200) - I'd rather have this included, but if you feel like it shouldn't be, I'll remove it.

Reply:
The idea is that kmod-usb-storage is a feature, and not necessary for boot/basic functionality. Some devices deviated from that, yes (don't know whether accidentally or intentionally).

I feel like it shouldn't be included, but if you have a strong tendency otherwise, I won't overrule you there.

Reply:
I added Arduino Yun support so I can explain why I included kmod-usb-storage for it.

USB port can be used for general usages, like printer, ethernet adapter, wifi dongle, etc. usb-storage does not cover all usages and can be dropped from default packages IMO.
In Yun's case, it has a micro SD card slot and the slot is connected to SoC via internal USB hub. SD card slot is only for storage, right? I think storage is the "basic" functionality of the slot, so I included usb-storage (and block-mount.)

Just my 2 cents.

Reply:
Thanks @mans0n for dropping by and detailing how the Yun is built - I wasn't aware that the SD-card slot is essentially a USB card reader. I've already dropped the usb-storage module anyway. :)